lib/ansible/modules/packaging/os/rpm_key.py

```diff
@@ -170,11 +170,15 @@ def execute_command(self, cmd):
         return stdout, stderr
 
     def is_key_imported(self, keyid):
-        cmd=self.rpm + ' -q  gpg-pubkey --qf "%{description}" | ' + self.gpg + ' --no-tty --batch --with-colons --fixed-list-mode -'
+        cmd = self.rpm + ' -q  gpg-pubkey'
+        rc, _, _ = self.module.run_command(cmd, use_unsafe_shell=True)
```

Comment:
`use_unsafe_shell` is not needed here, also can you please not use `_` (the reason being that if we ever get around to i18n'ing our code, we'll be using `_` to mark strings), so:

`rc, stdout, stderr = self.module.run_command(cmd)`

Reply:
I see, OK